# `no-loop-func`

<<<<<<< HEAD
Disallows function declarations that contain unsafe references inside loop statements.
=======
Disallow function declarations that contain unsafe references inside loop statements.
>>>>>>> fe

## Rule Details

This rule extends the base [`eslint/no-loop-func`](https://eslint.org/docs/rules/no-loop-func) rule.
It adds support for TypeScript types.

## How to Use

```jsonc
{
  // note you must disable the base rule as it can report incorrect errors
  "no-loop-func": "off",
  "@typescript-eslint/no-loop-func": ["error"]
}
```

## Options

See [`eslint/no-loop-func` options](https://eslint.org/docs/rules/no-loop-func#options).

<sup>

Taken with ‚ù§Ô∏è [from ESLint core](https://github.com/eslint/eslint/blob/main/docs/rules/no-loop-func.md)

</sup>
<<<<<<< HEAD
=======

## Attributes

- Configs:
  - [ ] ‚úÖ Recommended
  - [ ] üîí Strict
- [ ] üîß Fixable
- [ ] üí≠ Requires type information
>>>>>>> fe
