# `no-unused-expressions`

<<<<<<< HEAD
Disallows unused expressions.
=======
Disallow unused expressions.
>>>>>>> fe

## Rule Details

This rule extends the base [`eslint/no-unused-expressions`](https://eslint.org/docs/rules/no-unused-expressions) rule.
It adds support for optional call expressions `x?.()`, and directive in module declarations.

## How to Use

```jsonc
{
  // note you must disable the base rule as it can report incorrect errors
  "no-unused-expressions": "off",
  "@typescript-eslint/no-unused-expressions": ["error"]
}
```

## Options

See [`eslint/no-unused-expressions` options](https://eslint.org/docs/rules/no-unused-expressions#options).

<sup>

Taken with ‚ù§Ô∏è [from ESLint core](https://github.com/eslint/eslint/blob/main/docs/rules/no-unused-expressions.md)

</sup>
<<<<<<< HEAD
=======

## Attributes

- Configs:
  - [ ] ‚úÖ Recommended
  - [ ] üîí Strict
- [ ] üîß Fixable
- [ ] üí≠ Requires type information
>>>>>>> fe
